src/pages/projects/the-beginner-s-guide-to-vue-3/index.tsx

```diff
@@ -0,0 +1,230 @@
+import React, {FunctionComponent} from 'react'
+import {NextSeo} from 'next-seo'
+import Markdown from 'react-markdown'
+import {sanityClient} from 'utils/sanity-client'
+import groq from 'groq'
+import Image from 'next/image'
+import {find} from 'lodash'
+import {track} from 'utils/analytics'
+
+type LandingProps = {
+  course: any
+}
+
+const landingPage: FunctionComponent<LandingProps> = (props) => {
+  const {course} = props
+  const {title, ogImage, path, image, tags} = course
+
+  const introduction: any = find(course?.projects?.content, {
+    label: 'Introduction',
+  })
+  const projectBrief: any = find(course?.projects?.content, {
+    label: 'Project Brief',
+  })
+  const appRequirements: any = find(course?.projects?.content, {
+    label: 'App Requirements',
+  })
+  const appData: any = find(course?.projects?.content, {
+    label: 'App Data',
+  })
+  const developmentStandards: any = find(course?.projects?.content, {
+    label: 'Development Standards',
+  })
+  const stretchGoal: any = find(course?.projects?.content, {
+    label: 'Stretch Goal',
+  })
+  const appDesign: any = find(course?.projects?.content, {
+    label: 'App Design',
+  })
+  const Submission: any = find(course?.projects?.content, {label: 'Submission'})
+
+  const {productCard, productPage} = course?.projects
+
+  console.log(course)
+
+  return (
+    <>
+      <NextSeo
+        description={course.description}
+        title={title}
+        titleTemplate={'%s | egghead.io'}
+        twitter={{
+          site: `@eggheadio`,
+          cardType: 'summary_large_image',
+        }}
+        openGraph={{
+          title,
+          description: course.description,
+          site_name: 'egghead',
+          images: [
+            {
+              url: ogImage,
+            },
+          ],
+        }}
+      />
+      <div className="bg-gray-50 dark:bg-gray-900 sm:-my-5 -my-3 -mx-5 p-5">
+        <div className="mb-10 pb-10 xl:px-0 px-5 max-w-screen-xl mx-auto">
+          <div className="mt-10 mb-16 text-center">
+            <div className="mb-10">
+              <Image priority src={image} height="270" width="270" />
+            </div>
+            <p className="text-lg md:text-2xl leading-6 text-gray-500">
+              Portfolio Project
+            </p>
+            <h1 className="text-2xl md:text-4xl font-bold mt-2">
+              {course.projects.title}
+            </h1>
+          </div>
+          <div className="flex flex-col justify-center items-start mx-auto max-w-screen-md mb-16 w-full px-4">
+            <main className="prose dark:prose-dark prose-lg max-w-none w-full">
+              <Markdown>{introduction.text}</Markdown>
+              <Markdown>{projectBrief.text}</Markdown>
+              <section className="grid grid-cols-2 gap-4">
+                <Markdown className="col-span-1">
+                  {appRequirements.text}
```

Comment:
The Implicit grid comes to mind here. The order and height are off between the 3 components (title, list, code block). I would suggest one column instead of two here? 

![image](https://user-images.githubusercontent.com/57044804/120006494-c9b04b00-bf8d-11eb-974e-2b4806a06288.png)

<img width="467" alt="CleanShot 2021-05-28 at 08 07 00@2x" src="https://user-images.githubusercontent.com/57044804/120006606-e64c8300-bf8d-11eb-8d8e-950fd08e48a1.png">